src/components/form/form.js

```diff
@@ -24,7 +43,20 @@ class Form extends React.Component{
     console.log(e.target.value);
     this.setState({
       rest: e.target.value
+      
     })
+  }
+  handleSubmit = async (e) => {
+    e.preventDefault();
+
+    const response = await axios.get(this.state.apiUrl[0]);
+
+    let data = response.body;
+    let headerData = response.header;
+
+    this.props.updateResults(data, headerData);
+    this.setState({ input: '' });
+
   }
 
     render(){
```

Comment:
have line 65-78 as one form. also on line 65 add `onSubmit={this.handleSubmit}` inside the form tag